spec/requests/providers/applicants_spec.rb

```diff
@@ -21,9 +21,20 @@
         expect(response).to redirect_to(providers_legal_aid_applications_path)
       end
     end
+
+    context 'has already got applicant infro' do
+      let(:applicant) { create(:applicant) }
+      let(:laa) { create(:legal_aid_application, applicant: applicant) }
+      let(:get_request) { get "/providers/applications/#{laa.id}/applicant/edit" }
+
+      it 'display first_name' do
+        get_request
+        expect(response.body).to include(applicant.first_name)
```

Comment:
**[nit pick]** Consider using the `unescaped_response_body` helper here to ensure this test doesn't randomly fail if the applicant's first name has special character that are encoded differently.